2010-05-26  Aurelien Jarno  <aurelien@aurel32.net>

	* posix/regex_internal.h: include <sys/param.h> to get MAX 
	definition.

No topgit branch, TODO?

---
 posix/regex_internal.h |    1 +
 1 file changed, 1 insertion(+)

--- a/posix/regex_internal.h
+++ b/posix/regex_internal.h
@@ -26,6 +26,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/param.h>
 
 #if defined _LIBC
 # include <gnu/option-groups.h>
